--- ChangeLog	2025-01-26 11:18:40.011356002 -0500
+++ ChangeLog-spell-corrected	2025-01-26 11:18:32.437349916 -0500
@@ -35,7 +35,7 @@
 2025-01-18  R. Bernstein <rocky@users.noreply.github.com>
 
 	* mathics_scanner/mathics3_tokens.py, mathics_scanner/tokeniser.py,
-	pyproject.toml, test/test_codetokenize.py, test/test_tokeniser.py: 
+	pyproject.toml, test/test_codetokenize.py, test/test_tokeniser.py:
 	Add mathics tokens cli (#115) * command-line routine showing Mathics3 tokenization * Add --CodeTokenize option . Option --CodeTokenize (-C) on
 	mathics3-tokens shows tokens more like it is shown in CodeParser's
 	CodeTokenize.  * Remove line number in In[] and Out[]. We don't have a % retrieval
@@ -43,7 +43,7 @@
 
 2025-01-15  R. Bernstein <rocky@users.noreply.github.com>
 
-	* mathics_scanner/data/named-characters.yml: amslatex(Vertial) ->
+	* mathics_scanner/data/named-characters.yml: amslatex(Vertical) ->
 	\smallmid. more unicod refs.. (#118) Remove note 5 which is was wrong, and probably comes from the days
 	when I was combining operators with character symbols which is
 	wrong.  Possibly in the future, this will be reworked so Operators (like
@@ -65,7 +65,7 @@
 
 2025-01-08  rocky <rb@dustyfeet.com>
 
-	* mathics_scanner/generate/build_operator_tables.py: 
+	* mathics_scanner/generate/build_operator_tables.py:
 	operator-precedence -> operator-precedences Note: Mathics-core will be broken until this is synchronized.
 
 2025-01-08  rocky <rb@dustyfeet.com>
@@ -89,7 +89,7 @@
 
 2025-01-05  R. Bernstein <rocky@users.noreply.github.com>
 
-	* mathics_scanner/data/operators.yml, mathics_scanner/tokeniser.py: 
+	* mathics_scanner/data/operators.yml, mathics_scanner/tokeniser.py:
 	Correct FractionBox and other small changes (#113)
 
 2025-01-03  R. Bernstein <rocky@users.noreply.github.com>
@@ -132,7 +132,7 @@
 2024-12-03  R. Bernstein <rocky@users.noreply.github.com>
 
 	* mathics_scanner/data/named-characters.yml,
-	mathics_scanner/data/operators.yml, test/test_mathics_precedence.py: 
+	mathics_scanner/data/operators.yml, test/test_mathics_precedence.py:
 	operator tests yet again; update YAML comments... (#106) * Test_mathics_precedence needs not to fail work when mathics is   installed. Tweak failure messages * Update comments in character and operators YAML files.  * Remove Parse field in operators YML.
 
 2024-12-03  Juan Mauricio Matera <matera@fisica.unlp.edu.ar>
@@ -198,10 +198,10 @@
 	mathics_scanner/data/operators.yml,
 	mathics_scanner/generate/operator_csv_to_yml.py,
 	mathics_scanner/tokeniser.py: tokenizer gets no-meaning infix ops
-	from JSON (#87) * tokenizer gets no-meaning infix ops from JSON * Temporily test from operator-info-from-JSON branch * Function unicode change so as not to conflict with RightTeeArrow.
+	from JSON (#87) * tokenizer gets no-meaning infix ops from JSON * Temporally test from operator-info-from-JSON branch * Function unicode change so as not to conflict with RightTeeArrow.
 	Function unicode is a long arrow.  * Remove CSV to YML stuff. CSV is beyond hope of keeping in sync.  * Remove tokeniser import from __init__.py. Workaround for now. We
 	need this so we can create operator JSON without needing the JSON
-	table to be prevously around.  * Split typing changing variables like tokens and literal_tokens
+	table to be previously around.  * Split typing changing variables like tokens and literal_tokens
 	intoseparate variables for each type they can hold.  * Scanner uses operators json mm (#88, #89) * Change initialization of the tokenizer --------- Co-authored-by: Juan Mauricio Matera <matera@fisica.unlp.edu.ar>
 
 2024-11-23  rocky <rb@dustyfeet.com>
@@ -230,7 +230,7 @@
 	mathics_scanner/generate/build_operator_tables.py,
 	test/test_operators.py: Include no-meaning operators in JSON op
 	extraction (#85) Include no-meaning operators in JSON op extraction, separated by
-	affix: Infix, Postifx, or Prefix.  Also, update "meaningful" field comment at the top of the YAML file.  Add a unit test for operator-table consistency.
+	affix: Infix, Postfix, or Prefix.  Also, update "meaningful" field comment at the top of the YAML file.  Add a unit test for operator-table consistency.
 
 2024-11-18  R. Bernstein <rocky@users.noreply.github.com>
 
@@ -272,8 +272,8 @@
 2024-09-17  R. Bernstein <rocky@users.noreply.github.com>
 
 	* mathics_scanner/data/named-characters.yml,
-	mathics_scanner/data/operators.yml: Add rljacobson's heursitic on
-	Unicode (#78) * Add rljacobson's heursitic on Unicode * Some small corrections
+	mathics_scanner/data/operators.yml: Add rljacobson's heuristic on
+	Unicode (#78) * Add rljacobson's heuristic on Unicode * Some small corrections
 
 2024-09-09  R. Bernstein <rocky@users.noreply.github.com>
 
@@ -457,7 +457,7 @@
 2023-12-17  rocky <rb@dustyfeet.com>
 
 	* admin-tools/pyenv-versions,
-	mathics_scanner/data/named-characters.yml, test/test_tokeniser.py: 
+	mathics_scanner/data/named-characters.yml, test/test_tokeniser.py:
 	Administrivia: typos and bump versions
 
 2023-08-23  rocky <rb@dustyfeet.com>
@@ -560,7 +560,7 @@
 	test/test_general_yaml_sanity.py,
 	test/test_has_unicode_inverse_sanity.py,
 	test/test_table_consistency.py, test/test_unicode_equivalent.py: Go
-	over character tables * Separate entries with a blank line * Remove "unicode-equivalent" when it is the same as "ascii" Because if this wl_to_unicode() nees to pick use "acsii" when
+	over character tables * Separate entries with a blank line * Remove "unicode-equivalent" when it is the same as "ascii" Because if this wl_to_unicode() needs to pick use "ascii" when
 	"unicode-equivalent" there is no unicode-equivalent, but
 	has-unicode-inverse is set.  * Add test that unicode-equivalent isn't the same things as ascii
 
@@ -686,7 +686,7 @@
 
 	* mathics_scanner/data/named-characters.yml,
 	mathics_scanner/tokeniser.py: Change the precedence of `|->`
-	(`Function` symbol) to 800 so it isn't iterpreted as a `|` followed
+	(`Function` symbol) to 800 so it isn't interpreted as a `|` followed
 	by `->`
 
 2021-09-27  rocky <rb@dustyfeet.com>
@@ -785,18 +785,18 @@
 2021-06-29  rocky <rb@dustyfeet.com>
 
 	* mathics_scanner/data/named-characters.yml,
-	test/test_general_yaml_sanity.py, test/test_table_consistency.py: 
+	test/test_general_yaml_sanity.py, test/test_table_consistency.py:
 	Appease current checker...  The checker may change in the future.
 
 2021-06-29  rocky <rb@dustyfeet.com>
 
-	* mathics_scanner/data/named-characters.yml: Noe one more source
+	* mathics_scanner/data/named-characters.yml: We now have one more source
 
 2021-06-29  rocky <rb@dustyfeet.com>
 
 	* mathics_scanner/data/named-characters.yml,
 	mathics_scanner/generate/build_tables.py: Add ApplyTo and reqularize
-	unicode-equivalent Also adust Tilde
+	unicode-equivalent Also adjust Tilde
 
 2021-06-28  rocky <rb@dustyfeet.com>
 
@@ -817,7 +817,7 @@
 
 2021-06-27  rocky <rb@dustyfeet.com>
 
-	* CHANGES.rst, MANIFEST.in, mathics_scanner/version.py, setup.py: 
+	* CHANGES.rst, MANIFEST.in, mathics_scanner/version.py, setup.py:
 	Get ready for release 1.2.3
 
 2021-06-26  rocky <rb@dustyfeet.com>
@@ -907,7 +907,7 @@
 	mathics_scanner/tokeniser.py, mathics_scanner/version.py, setup.py,
 	test/test_has_unicode_inverse_sanity.py,
 	test/test_letterlikes_sanity.py, test/test_table_consistency.py,
-	test/test_translation_regressions.py, test/test_wl_to_ascii.py: 
+	test/test_translation_regressions.py, test/test_wl_to_ascii.py:
 	Blacken files
 
 2021-06-21  rocky <rb@dustyfeet.com>
@@ -948,7 +948,7 @@
 
 	* .github/workflows/osx.yaml, .github/workflows/ubuntu.yaml,
 	.github/workflows/windows.yml, mathics_scanner/characters.py,
-	mathics_scanner/version.py, requirements-dev.txt, setup.py: 
+	mathics_scanner/version.py, requirements-dev.txt, setup.py:
 	Administrivia & make ujson optional * It's not clear that pyston-2.2 support ujson so make it optional * In github workflows use requirements-dev.txt * revise requirement-dev to remove click (in setup.py) and add ujson * setup.py make ujson optional * version.py: bump to dev
 
 2021-05-14  rocky <rb@dustyfeet.com>
@@ -1074,7 +1074,7 @@
 2021-04-03  rocky <rb@dustyfeet.com>
 
 	* mathics_scanner/data/named-characters.yml,
-	mathics_scanner/tokeniser.py: Start RightComposition...  /* - still ned to complete tokenizer and parser, and operator parts
+	mathics_scanner/tokeniser.py: Start RightComposition...  /* - still need to complete tokenizer and parser, and operator parts
 
 2021-04-03  rocky <rb@dustyfeet.com>
 
@@ -1148,7 +1148,7 @@
 
 	* .github/workflows/osx.yaml, Makefile,
 	mathics_scanner/data/named-characters.yml, setup.py,
-	test/test_general_yaml_sanity.py, test/test_table_consistency.py: 
+	test/test_general_yaml_sanity.py, test/test_table_consistency.py:
 	Start tagging operators Note: use WL name when there is no conflict for name of symbol.  A number of other small corrections: - Mark some symbols which are invertable when they are. However   this should be redone and invertability should be detected   automatically.  - small correction in macos CI - AUTHORS.txt reduced to those who worked in this, rather than
 	  Mathics in general - in tests prefer top-level tests when that is possible (i.e. test
 	  is not parameterized) - use relative import as "test" can be ambiguous and lead to
@@ -1249,8 +1249,8 @@
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* README.rst, implementation.rst, mathics_scanner/characters.py: 
-	Added an awfull lot of documentation
+	* README.rst, implementation.rst, mathics_scanner/characters.py:
+	Added an awful lot of documentation
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
@@ -1282,7 +1282,7 @@
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* mathics_scanner/data/named-characters.yml, test/test_roundtrip.py: 
+	* mathics_scanner/data/named-characters.yml, test/test_roundtrip.py:
 	Fixed another errors in the YAML table
 
 2021-02-02  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
@@ -1305,7 +1305,7 @@
 
 2021-02-01  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* mathics_scanner/errors.py, mathics_scanner/tokeniser.py: 
+	* mathics_scanner/errors.py, mathics_scanner/tokeniser.py:
 	Documented stuff used by Mathics
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
@@ -1325,7 +1325,7 @@
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* test/test_general_yaml_sanity.py, test/test_letterlikes_sanity.py: 
+	* test/test_general_yaml_sanity.py, test/test_letterlikes_sanity.py:
 	Improved the general tests for YAML
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
@@ -1354,22 +1354,22 @@
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* test/test_has_unicode_inverse_sanity.py, test/test_roundtrip.py: 
+	* test/test_has_unicode_inverse_sanity.py, test/test_roundtrip.py:
 	Fixed typos
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* test/test_has_unicode_inverse_sanity.py, test/test_roundtrip.py: 
+	* test/test_has_unicode_inverse_sanity.py, test/test_roundtrip.py:
 	Improved the sanity checks for has-unicode-inverse
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* test/test_has_unicode_inverse_sanity.py, test/test_roundtrip.py: 
+	* test/test_has_unicode_inverse_sanity.py, test/test_roundtrip.py:
 	Fixed and improved the roundtrip tests
 
 2021-01-31  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* test/test_has_unicode_inverse_sanity.py, test/test_roundtrip.py: 
+	* test/test_has_unicode_inverse_sanity.py, test/test_roundtrip.py:
 	Added sanity check for has-unicode-inverse and fixed the reound trip
 	tests
 
@@ -1421,7 +1421,7 @@
 
 2021-01-27  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* mathics_scanner/build_tables.py: Improved the algorith that
+	* mathics_scanner/build_tables.py: Improved the algorithm that
 	resolves the appropriate plain-text representation of a given
 	character
 
@@ -1577,15 +1577,15 @@
 
 2021-01-19  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* setup.py: Tryed to fix the installation script
+	* setup.py: Tried to fix the installation script
 
 2021-01-19  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* Makefile, setup.py: Tryed to fix CI
+	* Makefile, setup.py: Tried to fix CI
 
 2021-01-19  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* setup.py: Tryed to fix the installation errors
+	* setup.py: Tried to fix the installation errors
 
 2021-01-19  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
@@ -1763,5 +1763,4 @@
 
 2021-01-18  Pablo Emilio Escobar Gaviria <pablo-escobar@riseup.net>
 
-	* Removed unnecessary comments (the old code wass commented)
-
+	* Removed unnecessary comments (the old code was commented)
