--- ChangeLog	2025-01-25 17:55:55.908438689 -0500
+++ ChangeLog.spell-corrected	2025-01-25 17:56:29.367444130 -0500
@@ -776,11 +775,11 @@
 2021-06-29  rocky <rb@dustyfeet.com>

-	* mathics_scanner/data/named-characters.yml: Noe one more source
+	* mathics_scanner/data/named-characters.yml: We now have one more source

 2021-06-29  rocky <rb@dustyfeet.com>

 	* mathics_scanner/data/named-characters.yml,
 	mathics_scanner/generate/build_tables.py: Add ApplyTo and reqularize
-	unicode-equivalent Also adust Tilde
+	unicode-equivalent Also adjust Tilde

 2021-06-28  rocky <rb@dustyfeet.com>
